src/Container/AgFeatureGrid/AgFeatureGrid.spec.jsx

```diff
@@ -0,0 +1,412 @@
+
+    wrapper.unmount();
+
+    expect(mapUnSpy).toHaveBeenCalledTimes(2);
+    expect(mapUnSpy).toHaveBeenCalledWith('pointermove', onPointerMove);
+    expect(mapUnSpy).toHaveBeenCalledWith('singleclick', onMapSingleClick);
+
+    mapUnSpy.mockReset();
+    mapUnSpy.mockRestore();
+  });
+
+  it('generates the column definition out of the given features and takes attributeBlacklist into account', () => {
+    const wrapper = TestUtil.mountComponent(AgFeatureGrid, {map, features});
+
+    const got = wrapper.instance().getColumnDefs();
+
+    const exp = [{
+      'field': 'id',
+      'headerName': 'id'
+    }, {
+      'field': 'name',
+      'headerName': 'name'
+    }];
+
+    expect(got).toEqual(exp);
+
+    wrapper.setProps({
+      attributeBlacklist: ['id']
+    });
+
+    const gotBlacklisted = wrapper.instance().getColumnDefs();
+
+    const expBlacklisted = [{
+      field: 'name',
+      headerName: 'name'
+    }];
+
+    expect(gotBlacklisted).toEqual(expBlacklisted);
+  });
+
+  it('generates the appropriate data to render', () => {
+    const wrapper = TestUtil.mountComponent(AgFeatureGrid, {map, features});
+
+    const got = wrapper.instance().getRowData();
+
+    const expRows = [{
+      id: 1,
+      name: 'Shinji Kagawa'
+    }, {
+      id: 2,
+      name: 'Marco Reus'
+    }, {
+      id: 3,
+      name: 'Roman Weidenfeller'
+    }];
+
+    expRows.forEach((row, idx) => {
+      expect(row.id).toEqual(got[idx].id);
+      expect(row.name).toEqual(got[idx].name);
+    });
+  });
+
+  it('fits the map to show all given features', () => {
+    const wrapper = TestUtil.mountComponent(AgFeatureGrid, {map, features});
+
+    const mapViewFitSpy = jest.spyOn(map.getView(), 'fit');
+
+    wrapper.instance().zoomToFeatures(features);
+
+    let featGeometries = [];
+    features.forEach(feature => {
+      featGeometries.push(feature.getGeometry());
+    });
+
+    expect(mapViewFitSpy).toHaveBeenCalledWith(new OlGeomGeometryCollection(featGeometries).getExtent());
+  });
+
+  it('highlights all given features', () => {
+    const wrapper = TestUtil.mountComponent(AgFeatureGrid, {map, features});
+
+    wrapper.instance().highlightFeatures(features);
+
+    features.forEach(feature => {
+      expect(feature.getStyle()).toEqual(wrapper.prop('highlightStyle'));
+    });
+  });
+
+  // it('unhighlight all given features, but takes selection into account', () => {
+  //   const wrapper = TestUtil.mountComponent(AgFeatureGrid, {map, features});
+  //   const selectedFeatureUid = features[0].ol_uid;
+  //
+  //   wrapper.setState({selectedRowKeys: [selectedFeatureUid]});
+  //   wrapper.instance().unhighlightFeatures(features);
+  //
+  //   features.forEach(feature => {
+  //     if (feature.ol_uid === selectedFeatureUid) {
+  //       expect(feature.getStyle()).toEqual(wrapper.prop('selectStyle'));
+  //     } else {
+  //       expect(feature.getStyle()).toBe(null);
+  //     }
+  //   });
+  // });
```

Comment:
Can we activate the commented out tests now? Here and below.

Reply:
Tried this, but unfortunately it's not that easy. I added a comment in #432.